Source/WebCore/rendering/EventRegion.cpp

```diff
@@ -253,8 +249,9 @@ bool EventRegionContext::shouldConsolidateInteractionRegion(RenderObject& render
         }
 
         // We found a region nested inside a container candidate for removal, flag it for removal.
-        if (m_containerRemovalCandidates.contains(ancestorElementIdentifier)) {
-            m_containerRemovalCandidates.remove(ancestorElementIdentifier);
+        auto removalCandidatesIterator = m_containerRemovalCandidates.find(ancestorElementIdentifier);
+        if (removalCandidatesIterator != m_containerRemovalCandidates.end()) {
+            m_containerRemovalCandidates.remove(removalCandidatesIterator);
```

Comment:
Nit: you could also check that `remove()` changed anything.

if (m_containerRemovalCandidates.remove(ancestorElementIdentifier))
    m_containersToRemove.add(ancestorElementIdentifier);